environment.yml

```diff
@@ -9,12 +9,12 @@ dependencies:
   - openssh
   - pip
   - pip:
-    - git+https://github.com/MAAP-Project/stac_ipyleaflet.git@0.3.3#egg-info=stac_ipyleaflet
+    - git+https://github.com/MAAP-Project/stac_ipyleaflet.git@0.3.6
     - jupyter-sshd-proxy
     - jupyterlab-bxplorer
 variables:
-  TITILER_STAC_ENDPOINT: 'https://staging-stac.delta-backend.com'
-  TITILER_ENDPOINT: 'https://staging-raster.delta-backend.com'
+  TITILER_STAC_ENDPOINT: 'https://openveda.cloud/api/stac'
```

Comment:
This TITILER_STAC variable is odd. We used to have a `/api/raster/stac` endpoint but that's now deprecated and we have a different prefix. Wonder if it's supposed to refer to that endpoint or if its just looking for the stac catalog (i.e. /api/stac).

Reply:
The variables seem to be defined for use in `stac_ipyleaflet`. Looking at [the references in the code base](https://github.com/search?q=repo%3AMAAP-Project%2Fstac_ipyleaflet%20TITILER_STAC_ENDPOINT&type=code), here's an example of it's use: `assets_endpoint = f"{self.titiler_stac_endpoint}/mosaicjson/{str_bounds}/assets?url={mosaic_url}/mosaicjson"` where `self.titiler_stac_endpoint` is set using `TITILER_STAC_ENDPOINT`. Does that help provide info about what the variable should be set as? I could look more if not, I just don't have the available endpoints memorized at this point.

Reply:
Ok this makes sense, thanks for providing that. So the old [raster_api]/stac endpoints should be updated to the new titiler-pgstac endpoints which follow this pattern -> /collections/{collection_id}/items/{item_id} 
I think the assets example above would now be:
/collections/{collection_id}/items/{item_id}/assets

Reply:
you can see the new endpoints here: https://openveda.cloud/api/raster/docs

Reply:
I searched `TITILER_STAC_ENDPOINT` here in pangeo-notebook-veda-image and in veda-docs and couldn't find any usage. 

The only case I can find is in `stac_ipyleaflet` where looks like MAAP uses it for one of their titilers https://github.com/MAAP-Project/stac_ipyleaflet/blob/main/README.md?plain=1#L60

@jjfrench @emmalu do you think veda can just use the default veda titiler here (openveda.cloud/api/raster)?

Reply:
I meant to open an issue for this, but perhaps we could have the conversation here. Zooming out - what's the plan for https://github.com/MAAP-Project/stac_ipyleaflet? I think we could have some CI to ensure the VEDA tutorial always works if we plan to maintain the widget, but if the functionality is likely to be built into other tools instead (e.g., lonboard), it would likely be better to remove these env variables and the notebook from the VEDA docs.

Reply:
TITILER_STAC_ENDPOINT comes from MAAP where titiler-stac.maap-project.org was a Titiler with the /stac endpoint - a bit dated now since these endpoints have moved.

We plan on removing titiler-stac.maap-project.org and consolidate our titilers, which would mean making changes to stac-ipyleaflet because the endpoints it uses are different/older. So @anayeaye , if this is setting the TITILER_STAC tiler to use for stac-ipyleaflet, I think that using openveda.cloud/api/raster likely wouldn't work with all functionalities because it's manually adding /stac and /cog onto the endpoints

Might be easiest to remove any tests using stac-ipyleaflet for now if you've already updated all of your titiler instances

Reply:
Just dropping this semi-related issue so it is findable if needed https://github.com/MAAP-Project/stac_ipyleaflet/issues/130